source/_integrations/ecoforest.markdown

```diff
@@ -26,8 +26,9 @@ There is currently support for the following device platforms within Home Assist
 
 ## Prerequisites
 
-To configure the Ecoforest integration you will need to enter your Ecoforest credentials which are the same ones you would use with the manufacturer app. The image below shows how to obtain the credentials:
+To configure the Ecoforest integration you will need to enter your Ecoforest host and credentials which are the same ones you would use with the manufacturer app. The image below shows how to obtain the credentials:
 
+- Host: it must start with http://ip_address or https://ip_address.
```

Comment:
That seems like a bug? If a host was requested, it should accept hosts. Otherwise, it would have been a URL...

../Frenck